lib/js/utils.js

```diff
@@ -0,0 +1,292 @@
+/**
+ * @typedef {Object} Node
+ * @property {string} type
+ */
+
+module.exports = {
+  /**
+   * Returns first expression in BlockStatement
+   *
+   * @param {?Array} body
+   * @returns {?Node}
+   */
+  getFirstExpression: function(body){
+    if (body)
+      return body[0] && body[0].expression || body[0];
+
+    return null;
+  },
+  /**
+   * Returns node body if its has a body
+   *
+   * @param {Node} node
+   * @returns {?Array}
+   */
+  getBody: function(node){
+    if (node.type === 'BlockStatement' || node.type === 'ClassBody' || node.type === 'Program')
+      return node.body;
+    else if(this.isTypeBlock(node.type) || this.isTypeClass(node.type) || this.isTypeFunction(node.type))
+      return node.body.body;
+
+    return null;
+  },
+  /**
+   * Create literal from string, number or boolean
+   *
+   * @param {string|number|boolean} value
+   * @returns {?Literal}
+   */
+  createLiteral: function(value){
+    var valueType = typeof value;
+
+    if (valueType === 'string' || valueType === 'number' || valueType === 'boolean')
+    {
+      return {
+        type: 'Literal',
+        value: value,
+        raw: valueType === 'string' ? '"' + value + '"' : String(value)
```

Comment:
For string you need something like:
```js
value.replace(/\\/g, '\\\\').replace(/\r/g, '\\r').replace(/\n/g, '\\n').replace(/"/g, '\\"')....
```
I think it's better to use `JSON.stringify()` here